topology/topologymodule.cpp

```diff
@@ -1239,6 +1243,39 @@ TopologyModule::Cvdict_MFunction::execute( SLIInterpreter* i ) const
   i->EStack.pop();
 }
 
+
+void
+TopologyModule::SelectNodesByMask_L_M_d_dFunction::execute(
+  SLIInterpreter* i ) const
+{
+  i->assert_stack_load( 4 );
+
+  const index& layer_gid = getValue< long >( i->OStack.pick( 0 ) );
+  MaskDatum mask = getValue< MaskDatum >( i->OStack.pick( 1 ) );
+  double xpos = getValue< double >( i->OStack.pick( 2 ) );
+  double ypos = getValue< double >( i->OStack.pick( 3 ) );
```

Comment:
Change to `anchor` as array instead of `xpos` and `ypos`. Also, I think the following order of arguments would be more logical: layer - anchor - mask, reading it as "from layer around anchor select all in mask".